### PR TITLE
Blender setup++

### PR DESCRIPTION
**Pull request by pmikolajczyk41**
Improve setting up the local environment:
 - two token contracts
 - two blender users

### CODE REVIEW

**Comment by pmikolajczyk41** [comment]
inb4: we cannot use `//Bob`, because `run_nodes.sh` endows only `//Alice` and `//0|1|2|3` with money

File: contracts/setup_blending.sh [hunk 1]
```diff
@@ -2,24 +2,53 @@
...
-TOKENS=1000
+# Actors
+CONTRACTS_ADMIN=//Alice
+DAMIAN=//0
+HANS=//1
+DAMIAN_ACCOUNT=5D34dL5prEUaGNQtPPZ3yN5Y6BnkfXunKXXz6fo7ZJbLwRRH
+HANS_ACCOUNT=5GBNeWRhZc2jXu7D55rBimKYDk8PGk8itRYFTPfC8RJLKG5o
+
+# Token economics
+TOTAL_TOKEN_ISSUANCE=2000
```
Comment (maciejzelaszczyk) [inline contracts/setup_blending.sh:15]: Would change this to `TOKEN_ISSUANCE_PER_CONTRACT`.

Comment (pmikolajczyk41) [reply contracts/setup_blending.sh]: sure, changed

**Review by maciejzelaszczyk — APPROVED** [review]
One nit. Apart from that, great effort :)

**Pull request merged**